docs/manual.rst

```diff
@@ -479,11 +479,20 @@ The file format is a bencoded dictionary containing the following fields:
 |                          | for, in case it is a v2 or v1+v2-hybrid torrent. This is a   |
 |                          | 32 byte SHA-256 hash of the info section of the torrent.     |
 +--------------------------+--------------------------------------------------------------+
-| ``pieces``               | A string with piece flags, one character per piece.          |
+| ``pieces``               | **deprecated**. This field is no longer written, but can     |
+|                          | can still be read.                                           |
+|                          | A string with piece flags, one character per piece.          |
 |                          | Bit 1 means we have that piece.                              |
 |                          | Bit 2 means we have verified that this piece is correct.     |
 |                          | This only applies when the torrent is in seed_mode.          |
 +--------------------------+--------------------------------------------------------------+
+| ``pieces2``              | A string where each bit indicates whether we have the piece  |
+|                          | or not.                                                      |
++--------------------------+--------------------------------------------------------------+
+| ``verified``             | A string where each bit indicates whether the piece has been |
```

Comment:
Why not re-use `pieces2`. IIRC, it isn't used in `seed-mode`.

Reply:
I think it would be risky to make the semantics of this field depend on a different flag (i.e. seed mode)

Reply:
>I think it would be risky to make the semantics of this field depend on a different flag (i.e. seed mode)

But in fact it is. Unless I'm wrong and there is existing case when both pieces and verified are supposed to be used simultaneously.
However, as you wish.

Reply:
yes, I don't think `pieces` and `verified` are used at the same time. At least it wouldn't make sense. however, there is some fault tolerance in giving them different names in case something goes wrong.